### ISSUE TITLE
Can't reorder palettes with keyboard

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#32767]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

There does not seem to be an accessible way to re order the titles in the palette list (currently it seems to be only via dragging with the mouse).  Suggest that the application key should be able to bring up a context menu here (same as right clicking) to include delete/hide, move up, move down and anything else that can usually be accessed with a right click or clicking settings dots.

STEPS:
1.  Navigate to the palette list.
2. Press application key (or Shift F10) and note nothing happens.
3. Press the number pad multiply to emulate a right mouse click.  This does seem to bring up a context menu, but it is for the note previously focused on the score and unrelated to the palette title navigated to.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.6.5

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible
